otherlibs/dune-site/test/run_2_9.t

```diff
@@ -181,7 +181,7 @@ Test with an opam like installation
     "_build/install/default/lib/a/a__.cmx"
     "_build/install/default/lib/a/a__.ml"
     "_build/install/default/lib/a/a__Sites.cmi"
-    "_destdir/_destdir/lib/a/a__Sites.cmt"
+    "_build/install/default/lib/a/a__Sites.cmt"
```

Comment:
Is this change expected?

Reply:
Yes, I put the reason in the commit message for squashing:

> the source is different because since cmt files are compressed in 5.1, they can be copied straight from the build directory (the placeholder is not detected)